src/drivers/config/conf_usb.h

```diff
@@ -46,7 +46,11 @@
 //	(USB_CONFIG_ATTR_REMOTE_WAKEUP|USB_CONFIG_ATTR_BUS_POWERED)
 
 // USB Device string definitions (Optional)
+#ifdef BOOTLOADER 
 #define  USB_DEVICE_MANUFACTURE_NAME      "Digital Bitbox, powered by ATMEL"

```

Comment:
btw: is the powered by ATMEL required? I think it will be visible in USB front ends like apples system-profiler, etc.